src/client/Microsoft.Identity.Client/Internal/Logger/LoggerHelper.cs

```diff
@@ -78,25 +78,25 @@ public static string GetClientInfo(string clientName, string clientVersion)
             return "Unknown SKU";
         });
 
-        public static string FormatLogMessage(string messageWithPii, string messageScrubbed, bool piiEnabled, string correlationId, string clientInformation)
+        public static string FormatLogMessage(string message, bool piiEnabled, string correlationId, string clientInformation)
         {
-            bool messageWithPiiExists = !string.IsNullOrWhiteSpace(messageWithPii);
-            // If we have a message with PII, and PII logging is enabled, use the PII message, else use the scrubbed message.
-            bool isLoggingPii = messageWithPiiExists && piiEnabled;
-            string messageToLog = isLoggingPii ? messageWithPii : messageScrubbed;
+            //bool messageWithPiiExists = !string.IsNullOrWhiteSpace(messageWithPii);
```

Comment:
if you comment this out, aren't you breaking existing logging functionality ? If not, please remove dead code.

Reply:
dead code